er3t/util/util.py

```diff
@@ -1,13 +1,15 @@
 import os
 import sys
 import csv
+import math
 import shutil
 import datetime
 import time
 import importlib
 import requests
 import urllib.request
 from io import StringIO
+import geopy.distance as gdist
```

Comment:
If `geopy` is only used to calculate distance between two locations, `cartopy` should contain some similar functionaries (I will look into this). We should avoid introducing new dependent libraries (unless we really have to) to keep `er3t` compact and lightweight.